Clarify granular permissions opt-in is irreversible for read-only users (#9694)
fishtown-analytics/docs.getdbt.com: website/docs/docs/platform/manage-access/about-access.md

<LicenseOverrideNote />

> #### Enable granular permissions for Read-Only users

Reviewer: Hiya Mirna

i might have missed this so apologies in advance, but I couldn’t find that control in my account, and this [slack](https://dbt-labs.slack.com/archives/C02N953LRDF/p1784294023020019) describe mentions mid-migration from the granular-read-only-permissions flag to a self-serve account setting (not fully rolled out everywhere yet), plus enterprise entitlement work.

Author: yes i believe it's being worked on now and this pr is setting it up. figma also https://www.figma.com/design/QYgO1bcKM4p4eFKiv9bqFn/Account-settings?node-id=706-27630&p=f&t=6p3rVKWPWGA00wwF-0

Reviewer: Thanks Mirna — that helps. If the Account settings control is still rolling out / not visible on all accounts yet, maybe add a short availability note on Enable granular permissions for Read-Only users so customers aren’t looking for a button that isn’t there yet? (non-block)

> Granular permissions let you restrict which projects Read-Only users can access, allowing you to set permissions by group rather than applying them to the entire account.

Reviewer 2: Would modify this sentence for clarity, something like "rather than applying the default permissions for Read-Only licenses."

> Granular permissions let you restrict which projects Read-Only users can access, allowing you to set permissions by group rather than applying them to the entire account.

Author: ```suggestion
Granular permissions let you restrict which projects Read-Only users can access, allowing you to set permissions by group rather than applying the default permissions for Read-Only licenses.
```

Key things to know before enabling:
- Access stays Read-Only: This setting controls access to projects, not permissions (users remain Read-Only).
- It’s permanent: Enabling this setting is a one-time, irreversible change.
- Prepare first: Ensure all Read-Only users are assigned to their respective groups before enabling, or they may lose access to their projects.

To turn on granular permissions:

1. Go to **Account settings**.
2. Click **Enable granular permissions**.

> 3. Check the box acknowledging that this setting can't be reversed once enabled.

Reviewer: ```suggestion
3. Select the checkbox acknowledging that this setting can't be reversed once enabled.
```

Reviewer: Hiya Mirna

Just amended this to align with the style guide

<LicenseOverrideNote />

#### Enable granular permissions for Read-Only users

Granular permissions let you restrict which projects Read-Only users can access, allowing you to set permissions by group rather than applying the default permissions for Read-Only licenses.

> New accounts don't need this setting — Read-Only users on new accounts get granular permissions by default.

Author: ```suggestion
Note that new accounts don't need this setting as Read-Only users on new accounts get granular permissions by default.
```

Key things to know before enabling:
- Access stays Read-Only: This setting controls access to projects, not permissions (users remain Read-Only).

> - It’s permanent: Enabling this setting is a one-time, irreversible change.

Reviewer: ```suggestion
- It's permanent: Enabling this setting is a one-time, irreversible change.
```
